src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Collection;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.Optional;
```

Comment:
Will be better for java.util.Optional import to be placed before java.util.Set to follow the coding convention.